package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@trili/tezos-provider",
+  "description": "Tezos Provider for WalletConnect Protocol",
+  "version": "1.0.3",
+  "author": "Trilitech TriliTech <contact@trili.tech> (https://trili.tech)",
+  "homepage": "https://trili.tech",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/trilitech/tezos-connect",
+    "directory": "."
+  },
+  "license": "MIT",
+  "type": "module",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.es.js",
+  "unpkg": "dist/index.umd.js",
+  "types": "dist/types/index.d.ts",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "wallet",
+    "walletconnect"
+  ],
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build": "tsup-node",
+    "test:pre": "rm -rf ./test/test.db",
+    "test:run": "jest",
+    "test": "yarn test:pre; yarn test:run",
+    "prettier": "prettier --check '{src,test}/**/*.{js,ts,jsx,tsx}' --write",
+    "lint": "eslint -c 'eslint.config.mjs' --fix './src/**/*.ts'",
```

Comment:
IIRC eslint 9 (which config you are using) has the list of files already.
no need to use quotes